bin/build-changelog.sh

```diff
@@ -1,15 +1,26 @@
 #!/usr/bin/env bash
 
+REPLACE_START_RAW='s/{%/{% raw %}{%{% endraw %}/g'
+REPLACE_END_RAW='s/{{/{% raw %}{{{% endraw %}/g'
+ESCAPE_QUOT='1 s/"/\&quot;/g'
+ESCAPE_LT='1 s/</\&lt;/g'
+ESCAPE_GT='1 s/>/\&gt;/g'
+
 cd docs
 cp ../CHANGELOG.md source/tutorials/changelog.md
-sed -i \
-  -e 's/{%/{% raw %}{%{% endraw %}/g' \
-  -e 's/{{/{% raw %}{{{% endraw %}/g' \
-  -e '1 s/"/\&quot;/g' \
-  -e '1 s/</\&lt;/g' \
-  -e '1 s/>/\&gt;/g' \
-  source/tutorials/changelog.md
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    sed -i '' -e "$REPLACE_START_RAW" -e "$REPLACE_END_RAW" -e "$ESCAPE_QUOT" -e "$ESCAPE_LT" -e "$ESCAPE_GT" source/tutorials/changelog.md
+else
+    sed -i -e "$REPLACE_START_RAW" -e "$REPLACE_END_RAW" -e "$ESCAPE_QUOT" -e "$ESCAPE_LT" -e "$ESCAPE_GT" source/tutorials/changelog.md
```

Comment:
Could we somehow use variable or alias to make this simpler? Maybe something like

```bash
alias sedi="sed -i ''"
```

If we need to repeat all the args, I'm wondering if someone missed one of the them in future changes. Ultimately we'll need javascript build scripts but now I believe scripts are doing fine.

Reply:
I see try that here. Unfortunately it won't work in the case where `1i\` needs to have the newline after it to work on macOS though. But at least we can reduce the amount of duplication to just those two invocations if using an alias like you propose works.

Reply:
@harttle is this what you meant? https://github.com/TomasHubelbauer/liquidjs/pull/1

When I try this I get this error:
> sedi: command not found

I looked around and it seems the suggested solution is to use Bash functions instead: https://linuxize.com/post/how-to-create-bash-aliases/#creating-bash-aliases-with-arguments-bash-functions

WDYT we should do here? Try using Bash functions or keep the doubled up argument lists?

Reply:
Fixed! Using Bash functions works.